src/sentry/api/endpoints/dif_files.py

```diff
@@ -7,30 +7,31 @@
 
 from sentry.utils import json
 from sentry.api.serializers import serialize
-from sentry.api.bases.chunk import ChunkAssembleMixin
 from sentry.api.bases.project import ProjectEndpoint, ProjectReleasePermission
-from sentry.models import File, ChunkFileState, ProjectDSymFile
-
-
-class DifAssembleEndpoint(ChunkAssembleMixin, ProjectEndpoint):
+from sentry.models import ChunkFileState, ProjectDSymFile, FileBlob, \
+    FileBlobOwner, get_assemble_status, set_assemble_status
+
+
+def find_missing_chunks(organization, chunks):
+    """Returns a list of chunks which are missing for an org."""
+    if not chunks:
+        return []
+    blobs = list(FileBlob.objects.filter(checksum__in=chunks))
+    missing = set(chunks)
+    if not blobs:
+        return list(missing)
+    owned = set(x.blob_id for x in FileBlobOwner.objects.filter(
```

Comment:
nit: You can just do:

```python
owned = set(FileBlobOwner.objects.filter(
    blob__in=blobs,
).values_list('id', flat=True))
```

The key bit is the `values_list` since you don't actually need `FileBlobOwner` objects, you only need the ids.